src/js/common/schemaform/widgets/RadioWidget.jsx

```diff
@@ -23,7 +23,7 @@ export default function RadioWidget({
                 disabled={disabled}
                 onChange={_ => onChange(option.value)}/>
             <label htmlFor={`${id}_${i}`}>
-              {option.label}
+              {labels[option.value] || option.label}
```

Comment:
The built in StringField component creates an `enumOptions` array that zips up `enum` and `enumNames` as an array of objects. That's what we've been using and what this PR should override, so I'm having the labels object passed in through `uiSchema` take precedent.